packages/ws-client/src/data-source.ts

```diff
@@ -42,7 +43,7 @@ export type ClientWebsocketMessage =
 
 export type ServerWebsocketMessage = UpdateWebsocketMessage | EventResponseWebsocketMessage
 
-type Handler = () => void
+type Handler = (value: JSONValue) => void
```

Comment:
Aligned "createStore" with "Store" type which is now a Stream. We have a few different variants of get/subscribe throughout the codebase, so I was trying to unify in a way that they follow same shape.

Reply:
sweet!